CorePayments/src/main/java/com/paypal/android/corepayments/api/AuthenticationSecureTokenServiceAPI.kt

```diff
@@ -0,0 +1,65 @@
+package com.paypal.android.corepayments.api
+
+import androidx.annotation.RestrictTo
+import com.paypal.android.corepayments.APIClientError
+import com.paypal.android.corepayments.APIClientError.payPalSDKError
+import com.paypal.android.corepayments.APIRequest
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.HttpMethod
+import com.paypal.android.corepayments.PayPalSDKErrorCode
+import com.paypal.android.corepayments.RestClient
+import com.paypal.android.corepayments.base64encoded
+import org.json.JSONObject
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class AuthenticationSecureTokenServiceAPI(
+    private val coreConfig: CoreConfig,
+    private val restClient: RestClient = RestClient(configuration = coreConfig),
+) {
+    suspend fun getLowScopedAccessToken(): GetClientTokenResult {
```

Comment:
```suggestion
    suspend fun createLowScopedAccessToken(): GetClientTokenResult {
```
nit: Since we're technically creating a token rather than retrieving one that already exists.